lib/explorer/series.ex

```diff
@@ -3149,6 +3149,48 @@ defmodule Explorer.Series do
     Keyword.merge(defaults, opts, fn _key, _left, right -> right end)
   end
 
+  @doc """
+  Calculate the exponentially weighted moving average, given smoothing factor alpha.
+
+  ## Options
+
+    * `:alpha` - Optional smoothing factor which specifies the imporance given
+      to most recent observations. It is a value such that, 0 < alpha <= 1. Defaults to 0.5.
+
+    * `:adjust` - If set to true, it corrects the bias introduced by smoothing process.
+      Defaults to `true`.
+
+    * `:min_periods` - The number of values in the window that should be non-nil
+      before computing a result. Defaults to `1`.
+
+    * `:ignore_nulls` - If set to true, it ignore nulls in the calculation. Defaults to `true`.
```

Comment:
This option could be named `ignore_nils`, since we are talking from the Elixir user perspective.